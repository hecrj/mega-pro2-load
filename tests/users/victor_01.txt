7

20  
100 
40  
70
100
130
200 


14

5 1 4 0 3 0 0 6 2 0 0 0 7 11 14 0 0 0 8 10 0 12 0 0 9 0 13 0 0

40 
2 7 6

50 
1 7 

20
2 1 2

10
0

100
1 7

20
2 1 2

20
3 7 3 6

20
1 7

60
3 7 6 4

30
1 7

30
2 1 2

40
1 7

70 
1 7

10
1 3

-1 7 1 

-1 4 3 

-5 1 3 

-1 7 4 

-5 1 200 

-1 6 13 

-5 1 13 

-1 7 14

-1 3 16

-1 7 17

-1 1 20

-1 1 21

-1 2 23

-3 23 

-1 2 24

-3 25 

-1 5 55

-2 100 

-4 6 
1 7
0

-4 1 
1 7
0

-1 7 400

-6 
