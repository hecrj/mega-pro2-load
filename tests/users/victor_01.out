Peticion procesada y servidores
0 1      
5 7 8 9  
Peticion procesada y servidores
1 2
9        
Pelicula mas solicitada
4 1      
Peticion procesada y servidores
2 1          
5 7 8 10 12  
Pelicula mas solicitada
7 2      
Peticion procesada y servidores
3 2      
7 9      
Pelicula mas solicitada
7 2      
Peticion procesada y servidores
4 2        
1 2 5  
Peticion procesada y servidores
5 2      
7 14     
Peticion procesada y servidores
6 1      
5 8 9 13 
Peticion procesada y servidores
7 1      
6        
Peticion procesada y servidores
8 1      
6        
Peticion procesada y servidores
9 4     
11      
Servidores ocupados 
11 9 4
Peticion procesada y servidores
10 5 
6     
Servidores ocupados 
6 10 4
11 9 2     
Peticion procesada y servidores
11 0     
Peticiones pendientes   
0         
Peticion procesada y servidores
12 1      
1 2 5 6
